### PR TITLE
ADEN-13282 use Prebid price bucketing

### PR DESCRIPTION
**Pull request by mfarul**

### CODE REVIEW

**Review by dmnsobczak — APPROVED** [review]

File: src/ad-bidders/prebid/index.ts [hunk 1]
```diff
@@ -22,10 +22,40 @@ import { intentIQ } from './intent-iq';
...
+		{
+			max: 5,
+			increment: 0.05,
+		},
+		{
+			max: 10,
+			increment: 0.1,
+		},
+		{
+			max: 20,
```
Comment (dmnsobczak) [inline src/ad-bidders/prebid/index.ts:44]: Some display bidders use range up to 50$ - let's switch all of them to this extended range

File: src/ad-bidders/prebid/index.ts [hunk 2]
```diff
@@ -22,10 +22,40 @@ import { intentIQ } from './intent-iq';
...
+			increment: 0.5,
+		},
```
Comment (dmnsobczak) [inline src/ad-bidders/prebid/index.ts:46]:
```suggestion
		{
			max: 20,
			increment: 0.5,
		},
		{
			max: 50,
			increment: 1,
		},
```

File: src/ad-bidders/prebid/index.ts [hunk 3]
```diff
@@ -22,10 +22,40 @@ import { intentIQ } from './intent-iq';
 import { liveRamp } from './live-ramp';
 import { getWinningBid } from './prebid-helper';
 import { getSettings } from './prebid-settings';
-import { getPrebidBestPrice } from './price-helper';
+import { getPrebidBestPrice, roundBucketCpm } from './price-helper';
 
 const logGroup = 'prebid';
 
+const defaultGranularity = {
```
Comment (dmnsobczak) [inline src/ad-bidders/prebid/index.ts:29]:
```suggestion
const priceGranularity = {
```

File: src/ad-bidders/prebid/index.ts [hunk 4]
```diff
@@ -22,10 +22,40 @@ import { intentIQ } from './intent-iq';
...
+};
+
+const videoGranularity = {
+	buckets: defaultGranularity.buckets.concat([
+		{
+			max: 50,
+			increment: 1,
+		},
+	]),
+};
```
Comment (dmnsobczak) [inline src/ad-bidders/prebid/index.ts:57]:
```suggestion

```

File: src/ad-bidders/prebid/index.ts (hunk 1 above)
Comment (mfarul) [reply src/ad-bidders/prebid/index.ts:44]: done in 2nd PR, will be merged here

File: src/ad-bidders/prebid/index.ts (hunk 3 above)
Comment (mfarul) [reply src/ad-bidders/prebid/index.ts:29]: done in 2nd PR, will be merged here

**Review by dmnsobczak — DISMISSED** [review]

**Review by dmnsobczak — APPROVED** [review]

**Pull request merged**